src/apps/opening-hours/OpeningHours.tsx

```diff
@@ -0,0 +1,69 @@
+import ArrowLeft from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/ArrowLeft.svg";
+import ArrowRight from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/ArrowRight.svg";
+import * as React from "react";
+import OpeningHoursWeekList from "./OpeningHourWeekList";
+import {
+  getNextWeek,
+  getPreviousWeek,
+  getWeekDisplay
+} from "./OpeningHoursHelpers";
+import useOpeningHours from "./useOpeningHours";
+
+export type OpeningHoursProps = {
+  branchId: number;
+  initialDate: Date;
+};
+
+const OpeningHours: React.FC<OpeningHoursProps> = ({
+  branchId,
+  initialDate
+}) => {
+  const {
+    currentWeekDisplay,
+    currentWeekRange,
+    groupedOpeningHours,
+    navigateToPreviousWeek,
+    navigateToNextWeek,
+    isLoading
+  } = useOpeningHours(initialDate, branchId);
+
+  const nextWeek = getWeekDisplay(getNextWeek(currentWeekRange.start));
+  const previousWeek = getWeekDisplay(getPreviousWeek(currentWeekRange.start));
+
+  return (
+    <div className="opening-hours">
+      <div className="opening-hours__wrapper">
+        <div className="opening-hours__header">
+          <h2 className="opening-hours__heading">Åbningstider</h2>
+          <div className="opening-hours__navigation-controls">
+            <button
+              className="opening-hours__navigation-control"
+              aria-label={`Vis åbningstider for ${previousWeek}`}
+              onClick={navigateToPreviousWeek}
+              type="button"
+            >
+              <img src={ArrowLeft} alt="" />
```

Comment:
Please remember to make these texts translatable.

:+1: for adding a proper aria label to these buttons and using an empty alt attribute.

Reply:
added